<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolarConstantAzElPlotSettings</class>
 <widget class="QWidget" name="PolarConstantAzElPlotSettings">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="zoomLabel">
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="originYSpinBox">
       <property name="toolTip">
        <string>Sets the origin's y-coordinate</string>
       </property>
       <property name="suffix">
        <string notr="true"/>
       </property>
       <property name="prefix">
        <string>y=</string>
       </property>
       <property name="minimum">
        <number>-999999</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="zoomResetButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Resets the zoom to 100%</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="angleSpinBox">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="angleLabel">
       <property name="text">
        <string>Azimuth</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="originLabel">
       <property name="text">
        <string>Origin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="originResetButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Resets the origin to (0,0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="angleResetButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Resets azimuth to 0°</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="originXSpinBox">
       <property name="toolTip">
        <string>Sets the origin's x-coordinate</string>
       </property>
       <property name="prefix">
        <string>x=</string>
       </property>
       <property name="minimum">
        <number>-999999</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="zoomSpinBox">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>25</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
